test/assignment7-buildroot/buildroot-common-build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
@akshita-bhasin can we re-use the same build we use for [assignment 4](https://github.com/cu-ecen-5013/assignment-autotest/tree/master/test/assignment4-buildroot) and [assignment 5](https://github.com/cu-ecen-5013/assignment-autotest/blob/master/test/assignment5-buildroot/assignment-5-build.sh)?  What's the difference with this build script?

Reply:
I pushed a change to try this with the autograder